packages/riipen-ui/src/components/FormGroup.jsx

```diff
@@ -40,12 +40,12 @@ class FormGroup extends React.Component {
     const theme = this.context;
 
     return css.resolve`
-      .subtitle {
+      p.subtitle {
```

Comment:
what was the reason we needed to add this? just wondering. I just believe it would work either way

Reply:
I'm having an issue where the .title and .subtitle styling are getting overridden by .h3 and .root styling when applied to web.  You're right though, I should do some more digging and figure out the root cause of it.